### PR TITLE
Update example Prometheus config file.

### PR DESCRIPTION
**Pull request by juliusv**

### CODE REVIEW

File: examples/rack/prometheus.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+global:
+  scrape_interval: "15s"
+
+scrape_configs:
+  - job_name: "prometheus"
+    static_configs:
+      - targets:
+         - "localhost:9090"
```
Comment (grobie) [inline examples/rack/prometheus.yml]: space--

**Comment by grobie** [comment]
How did you find this? What a relict.

File: examples/rack/prometheus.yml (hunk 1 above)
Comment (juliusv) [inline examples/rack/prometheus.yml]: Thanks, fixed.

**Comment by juliusv** [comment]
Oh, just trying out the Ruby lib for a client and wanted to play with the example :)

**Pull request merged**